app/controllers/events_controller.rb

```diff
@@ -0,0 +1,45 @@
+class EventsController < ApplicationController
+  before_action :logged_in?, except: %i[show index]
+
+  def index
+    @events = Event.all
+  end
+
+  def new
+    if current_user.nil?
+      redirect_to '/login'
+    else
+      @event = current_user.events.build
+    end
+  end
+
+  def create
+    @event = Event.new(event_params)
+    @event.creator_id = current_user.id
+    respond_to do |format|
+      if @event.save
+        format.html { redirect_to events_path, notice: 'Event was successfully created.' }
+        # format.json { render :show, status: :created, location: @user }
+      else
+        format.html { render :new }
+        format.json { render json: @event.errors, status: :unprocessable_entity }
+      end
+    end
+  end
```

Comment:
Your create action is expected to use the #build method on association, i.e. `current_user.events.build(event_params)` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/build_method_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))